### PR TITLE
Add contact-form and validation 

### PR DESCRIPTION
**Pull request by khitermedachraf**
In this Pull Request, I worked with my coding partner for adding the CSS and HTML of the contact section, we build the contact form with all required validations and collect Data using [Formspree service](https://formspree.io/html), we use also  Flexbox to layout the elements  of the sections.

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -284,3 +287,96 @@ p {
...
+}
+
+.end-shape {
+  margin-top: 5em;
+  margin-bottom: 1em;
+}
+
+#form img {
+  position: absolute;
+  right: 0;
```
Comment (RolandM99) [inline style.css:385]: - [x] Good job so far by working on the project requirements as expected, but there is some issues that still exists. Please, kindly implement the your project exactly as the Figma design. This is to make sure that you follow the Figma design guideline and replicate the design to look almost identical to the original design.
### your design:
image
- Make sure that the text in textarea is in bold style.
- kindly, replace the image background on is exact place following the figma design.

**Review by RolandM99 — CHANGES_REQUESTED** [review]
Hi @khitermedachraf  and @ijeomaese , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


### To highlight 💯 
- Good commit message ✔️ 
- No Linter errors ✔️ 
- Follow Github flow ✔️ 

## Required Changes ♻️
_Check the comments under the review._

- [x] Make sure that your PR title is descriptive and telling us what changes are inside. For example, for this project you can use **"Add contact-form and validation"**. Please, Kindly fix it.
## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me on @Rolandm99 in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by khitermedachraf — COMMENTED** [review]
I need just to change the PR title

File: style.css (hunk 1 above)
Comment (khitermedachraf) [reply style.css:385]: okey, I just forget to add the id ="form" to html file, u add it to fix stylelint error about { no-descending-specificity}

Comment (khitermedachraf) [reply style.css:385]: Done! Thank you for the observation.

**Review by AmaduKamara — APPROVED** [review]
# Status: Approved 🟢🟢🟢 

Hey Team, 
Good job in making all the changes required by the previous reviewer. Keep rocking 💪💪
Your project is complete!  There is nothing else to say other than... it's time to merge it:
Congratulations! 🎉 

LetgoLetsGooooGIF


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**